ui/app/mirrors/create/handlers.ts

```diff
@@ -52,13 +52,20 @@ const validateQRepFields = (
   return true;
 };
 
+interface TableMapping {
+  sourceTableIdentifier: string;
+  destinationTableIdentifier: string;
+  partitionKey: string;
+}
 const reformattedTableMapping = (tableMapping: TableMapRow[]) => {
   const mapping = tableMapping.map((row) => {
-    return {
-      sourceTableIdentifier: row.source,
-      destinationTableIdentifier: row.destination,
-      partitionKey: row.partitionKey,
-    };
+    if (row.selected === true) {
```

Comment:
Need to avoid returning `undefined`, either with filter or by having `mapping = []` & pushing elements in a `for-of`